src/mixed-line-bar-chart/__integ__/mixed-line-bar-chart.test.ts

```diff
@@ -496,6 +496,33 @@ describe('Details popover', () => {
     })
   );
 
+  test(
+    'scrolls if necessary on click',
+    setupTest('#/light/bar-chart/drilldown', async page => {
+      await page.setWindowSize({ width: 360, height: 650 });
+      await page.windowScrollTo({ top: 150 });
+      const barChart = createWrapper().findBarChart();
+      const barGroup = barChart.findBarGroups().get(1).toSelector();
+      await page.clickBarGroup(barGroup);
+      await page.waitForVisible(barChart.findDetailPopover().toSelector());
+      await expect(page.getText(barChart.findDetailPopover().findHeader().toSelector())).resolves.toContain('Apr 2023');
+      await expect(page.getWindowScroll()).resolves.toEqual({ top: 0, left: 0 });
+    })
+  );
+
+  test(
+    'does not scroll on hover',
+    setupTest('#/light/bar-chart/drilldown', async page => {
+      await page.setWindowSize({ width: 360, height: 650 });
+      await page.windowScrollTo({ top: 150 });
+      const barChart = createWrapper().findBarChart();
+      const barGroup = barChart.findBarGroups().get(1).toSelector();
+      await page.hoverElement(barGroup);
```

Comment:
For a more solid test, you need to add the same assertion before and after hover interaction

```js
await expect(page.getWindowScroll()).resolves.toEqual({ top: 150, left: 0 });
```

Reply:
What could happen between `await page.windowScrollTo({ top: 150 });` and `await page.hoverElement(barGroup);` that could alter the scroll position? Or is there a reason why the first one would not set it correctly in the first place?

Reply:
`windowScrollTo` may hit the page limit and not scroll fully as expected.

More assertions is better than less assertions

Reply:
OK, added.